libevmcore/EVMSchedule.h

```diff
@@ -0,0 +1,73 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file EVMSchedule.h
+ * @author Gav <i@gavwood.com>
+ * @author Christian <c@ethdev.com>
+ * @date 2015
+ */
+
+#pragma once
+
+#include <libdevcore/Common.h>
+
+namespace dev
+{
+namespace eth
+{
+
+struct EVMSchedule
```

Comment:
This is the only thing we need from the whole ethcore / evm cruft. Please make sure to use the new version by greg that uses unsigned instead of u256. If that would be a single file in this directory, you can put it inside libevmasm.